server/sonar-web/src/main/js/apps/permission-templates/users-view.js

```diff
@@ -0,0 +1,49 @@
+import Modal from '../../components/common/modals';
+import '../../components/common/select-list';
+import './templates';
+
+export default Modal.extend({
+  template: Templates['permission-templates-users'],
+
+  onRender: function () {
+    this._super();
+    var searchUrl = baseUrl + '/api/permissions/template_users?ps=100&permission=' + this.options.permission +
+        '&templateId=' + this.options.permissionTemplate.id;
+    new window.SelectList({
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace by a standard call to the function. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=javascript%3AConstructorFunctionsForSideEffects)